share/csm_share/shr/shr_flux_mod.F90

```diff
@@ -267,7 +277,15 @@ real(R8)    :: psix2  ! stability function at ztref reference height
      if (mask(n) /= 0) then
     
         !--- compute some needed quantities ---
-        vmag   = max(umin, sqrt( (ubot(n)-us(n))**2 + (vbot(n)-vs(n))**2) )
+
```

Comment:
First - from examining the code carefully, this is actually not correct. It  ugust is in fact zero - but prec_gust is NOT (see line 1397 in seq_flux_mct.F90)
             prec_gust (n) = a2x%rAttr(index_a2x_Faxa_rainc,n)

This is then used in shr_flux_mct.F90 in the following

       !--- reverts to original formula if gust_fac=0 
        vmag   = max(umin, sqrt( (ubot(n)-us(n))**2 + (vbot(n)-vs(n))**2) +  ugust(min(prec_gust(n),6.94444e-4_R8)))

From this, acme is introducing new science in a non-backwards compatible manner. So we cannot leave this code as it is.

Second,  uGust and prec_gust are not namelist parameters - they are currently hard-wired in seq_flux_mct.F90 to zero. Furthermore, this line is commented out - which could easily be uncommented by a user 
seq_flux_mct.F90:!           !!uGust(n) = 1.5_r8*sqrt(uocn(n)**2 + vocn(n)**2) ! there is no wind gust data from ocn

What needs to be done in seq_flux_mct.F90 is the following:

if (cime_model == 'cesm') then
     uGust(n) = 0._r8
     prec_gust(n) = 0._r8
else
!    !!uGust(n) = 1.5_r8*sqrt(uocn(n)**2 + vocn(n)**2) ! there is no wind gust data from ocn 
     prec_gust(n) = a2x%rAttr(index_a2x_Faxa_rainc,n)
     uGust(n) = 0.0_r8
end if

**However, my strong preference would be to pass cime_model to shr_flux_mct.F90
and have the following lines appear in shr_flux_mct.F90**

        !--- vmag+ugust (convective gustiness) Limit to a max precip 6 cm/day = 0.00069444 mm/s.
        !--- reverts to original formula if gust_fac=0 
if (trim(cime_model) == 'acme') then
        vmag   = max(umin, sqrt( (ubot(n)-us(n))**2 + (vbot(n)-vs(n))**2) +  ugust(min(prec_gust(n),6.94444e-4_R8)))
else
        !--- vmag+ugust (convective gustiness) Limit to a max precip 6 cm/day = 0.00069444 mm/s.
        !--- reverts to original formula if gust_fac=0 
        vmag   = max(umin, sqrt( (ubot(n)-us(n))**2 + (vbot(n)-vs(n))**2)
end if

This would be much cleaner and would clarify how those arguments are used.
This would also eliminate having to introduce multiple if-blocks in seq_flux_mct.F90 to set ugust and prec_gust to zero depending on acme or cesm.